### PR TITLE
Multisig Inspection fixes

### PR DESCRIPTION
**Pull request by RodrigoAD**

### CODE REVIEW

File: packages-ts/gauntlet-terra-cw-plus/src/commands/inspect.ts [hunk 1]
```diff
@@ -47,6 +46,9 @@ export const fetchProposalState = (provider: LCDClient) => async (
     _queryMultisig({
       threshold: {},
     }),
+    _queryMultisig({
```
Comment (kylesmartin) [inline packages-ts/gauntlet-terra-cw-plus/src/commands/inspect.ts]: Is this a duplicate query?

**Review by kylesmartin — COMMENTED** [review]
Everything looks good outside of the one question I had.

**Comment by akhilchainani** [comment]
Looks good apart from Kyle's comment and the failing check (not sure why thats failing - I'm unable to see the error)

File: packages-ts/gauntlet-terra-cw-plus/src/commands/inspect.ts (hunk 1 above)
Comment (RodrigoAD) [reply packages-ts/gauntlet-terra-cw-plus/src/commands/inspect.ts]: It is, just removed it!

**Review by kylesmartin — APPROVED** [review]
LGTM

**Pull request merged**